SSR example doesn't work

**Resources:**
Before submitting an issue, please consult our [docs](https://stenciljs.com/).

**Stencil version:** (run `npm list @stencil/core` from a terminal/cmd prompt and paste output below):

```
scott@ax ~/workspace/my-app master
 [49] → npm list @stencil/core
@stencil/starter@0.0.1 /Users/scott/workspace/my-app
└── @stencil/core@0.0.5-17
```

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or http://ionicworldwide.herokuapp.com/

**Current behavior:**
Following the getting started guide from the docs, then pasting the SSR example from the docs into a file generates the following error:
```
scott@ax ~/workspace/my-app master
 [48] → node server.js
module.js:471
    throw err;
    ^

Error: Cannot find module '/Users/scott/workspace/my-app/node_modules/@stencil/core/bin/util'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at validateRendererConfig (/Users/scott/workspace/my-app/node_modules/@stencil/core/dist/server/index.js:5045:22)
    at Object.createRenderer (/Users/scott/workspace/my-app/node_modules/@stencil/core/dist/server/index.js:4930:5)
    at Object.<anonymous> (/Users/scott/workspace/my-app/server.js:9:24)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
```

**Expected behavior:**
I expect the example to work. 😉 

**Steps to reproduce:**
```
git clone https://github.com/ionic-team/stencil-starter.git my-app
cd my-app
git remote rm origin
npm install
npm start
npm run build
```
then paste the SSR example from the docs into `./server.js` and run `node server.js`

**Related code:**

```
var express = require('express');
var app = express();
var fs = require('fs');
var path = require('path');

var stencil = require('@stencil/core');

// Create the stencil SSR renderer
var renderer = stencil.createRenderer({
  rootDir: path.join(__dirname, './'),
  buildDir: path.join(__dirname, './www/build/'),
  namespace: 'app',
  logLevel: 'debug'
});

// host the build directory as static files
// so the app can pull client side scripts
app.use('/build', express.static('www/build'));

// If you want to use HTML5 style routing in your client, keep the catch-all route handler here,
// otherwise change it to a more specific route
app.get('/*', function (req, res, next) {
  console.log(`serve: ${req.url}`);

  var filePath = path.join(__dirname, 'www/index.html');

  fs.readFile(filePath, 'utf-8', (err, html) => {
    if (err) {
      console.error(err);
      res.send(err);
      return;
    }

    // Render the initial app content through Stencil
    renderer.hydrateToString({
      html: html,
      req: req,
      config: {}
    }, function(err, html) {
      if (err) {
        // Handle the error hydrating
        console.error(err);
        return res.sendStatus(500);
      }

      // Send the hydrated data back
      res.send(html);
    });
  });

});
```

**Other information:**
Big fan of what you guys are doing and excited to see where it goes!

--- a/src/server/renderer.ts
+++ b/src/server/renderer.ts
@@ -150,7 +150,7 @@ function validateRendererConfig(config: BuildConfig) {
     // if a logger was not provided then use the
     // defaul stencil command line logger found in bin
     const path = require('path');
-    const logger = require(path.join(__dirname, '../../bin/util')).logger;
+    const logger = require(path.join(__dirname, '../../cli/util')).logger;
     config.logger = new logger.CommandLineLogger({
       level: config.logLevel,
       process: process,

